prereise/gather/demanddata/transportation_electrification/generate_scaling_factors.py

```diff
@@ -129,17 +129,19 @@ def load_dot_vmt_per_capita(path):
     )
 
     return (
-        df_ua.squeeze().loc[lambda x: x != "[no data]"],
+        df_ua.squeeze().loc[~df_ua.index.str.contains("\[no data\]")],  # noqa: W605
         df_state.rename(index=state2abv).squeeze(),
     )
 
 
-def calculate_vmt_for_ua(census_ua, tht_ua):
+def calculate_vmt_for_ua(census_ua, tht_ua, tht_state):
     """Calculate the total annual Vehicle Miles Traveled (VMT) in urban areas
 
     :param dict census_ua: dictionary as returned by :func:`load_census_ua`
     :param pandas.Series tht_ua: vmt per capita in urban areas as returned by
         :func:`load_dot_vmt_per_capita`
+    :param pandas.Series tht_state: vmt per capita in state as returned by
```

Comment:
Same reasoning:
`:param pandas.Series tht_state: annual vmt per capita in state`

Reply:
Done